dht_test.go

```diff
@@ -1910,3 +1910,51 @@ func TestProtocolUpgrade(t *testing.T) {
 		t.Fatalf("Expected 'buzz' got '%s'", string(value))
 	}
 }
+
+func TestInvalidKeys(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	nDHTs := 2
```

Comment:
Worth having a function parameterized on this and then run the test with a 2 node and a >2 node instance?

Reply:
Probably not. This test just cares about having at least one other node so the routing table isn't empty.